run_test.sh

```diff
@@ -21,34 +21,41 @@ else
 fi
 
 # Some helpers
-dkms_status_grep_dkms_test() {
-    (dkms status | grep '^dkms_test/') || true
+dkms_status_grep_dkms_module() {
+    local module_name="$1"
+    (dkms status | grep "^${module_name}/") || true
```

Comment:
Two top-level nits:
 - let's squash the conf additions (patch 3) with the follow-up changes done here
 - split the no-op refactor separate changes as separate commit

No-op refactor changes:
 - `dkms_status_grep_dkms_test` -> `dkms_status_grep_dkms_module`, update callers
 - `clean_dkms_env`, `check_no_dkms_test` use `local -a test_modules=("dkms_test")` and a loop
 
Couple more notes coming shortly

Reply:
Done.